crypto/serializer/deserializer_pkey.c

```diff
@@ -112,7 +112,7 @@ static int deser_finalize_EVP_PKEY(OSSL_DESERIALIZER_INSTANCE *deser_inst,
      * contents entirely opaque to us, but may be passed to any provider
      * function that expects this (such as OSSL_FUNC_keymgmt_load().
      *
-     * This pointers are considered volatile, i.e. whatever they point at
+     * This pointers is considered volatile, i.e. whatever they point at
```

Comment:
Its either

These pointers are

OR

This pointer is ... whatever it points at